bluetooth/src/commonMain/kotlin/scanner/ScanningStateRepo.kt

```diff
@@ -125,6 +127,23 @@ open class ScanningStateImplRepo(
         }
     }
 
+    private suspend fun handlePairedDevice(event: Scanner.Event.PairedDevicesRetrieved) = takeAndChangeState(remainIfStateNot = ScanningState.Enabled::class) { state ->
+        val creators = event.deviceCreators.map {
+            {
+                val (deviceWrapper, connectionManagerBuilder) = it()
+                val rssi = Int.MIN_VALUE
+                val advertisementData = ConnectableEmptyAdvertisementData(deviceWrapper.name)
```

Comment:
Okay, now I see why but in this case "empty" is a little confusing.  Probably "Trivial" or "Simple" is a better choice?

Reply:
What about 'Unknown'? As we have no advertisement data for paired device, however we need to use it to create existing device (with connectable flag set to true in advance)

Reply:
I renamed it into `PairedAdvertisementData` as I have to provide `serviceUUIDs` as well